disnake/enums.py

```diff
@@ -299,1205 +277,175 @@ def __setattr__(self, name: str, value: Any) -> None:
-    """
-
     disabled = 0
-    """The guild does not have the content filter enabled."""
     no_role = 1
-    """The guild has the content filter enabled for members without a role."""
     all_members = 2
-    """The guild has the content filter enabled for every member."""
 
 
 class Status(str, Enum):
-    """Specifies a :class:`Member`\\'s status."""
-
     online = "online"
-    """The member is online."""
     offline = "offline"
-    """The member is offline."""
     idle = "idle"
-    """The member is idle."""
     dnd = "dnd"
-    """The member is on "Do Not Disturb"."""
     do_not_disturb = "dnd"
-    """An alias for :attr:`dnd`."""
     invisible = "invisible"
-    """The member is "invisible". In reality, this is only used in sending
-    a presence a la :meth:`Client.change_presence`. When you receive a
-    user's presence this will be :attr:`offline` instead.
-    """
     streaming = "streaming"
-    """The member is live streaming to Twitch.
-
-    .. versionadded:: 2.3
-    """
 
 
 class DefaultAvatar(int, Enum):
-    """Represents the default avatar of a Discord :class:`User`"""
-
     blurple = 0
-    """Represents the default avatar with the color blurple.
-    See also :attr:`Colour.blurple`
-    """
     grey = 1
-    """Represents the default avatar with the color grey.
-    See also :attr:`Colour.greyple`
-    """
     gray = 1
-    """An alias for :attr:`grey`."""
     green = 2
-    """Represents the default avatar with the color green.
-    See also :attr:`Colour.green`
-    """
     orange = 3
-    """Represents the default avatar with the color orange.
-    See also :attr:`Colour.orange`
-    """
     red = 4
-    """Represents the default avatar with the color red.
-    See also :attr:`Colour.red`
-    """
 
 
 class NotificationLevel(int, Enum):
-    """Specifies whether a :class:`Guild` has notifications on for all messages or mentions only by default.
-
-    .. container:: operations
-
-        .. versionadded:: 2.0
-
-        .. describe:: x == y
-
-            Checks if two notification levels are equal.
-        .. describe:: x != y
-
-            Checks if two notification levels are not equal.
-        .. describe:: x > y
-
-            Checks if a notification level is higher than another.
-        .. describe:: x < y
-
-            Checks if a notification level is lower than another.
-        .. describe:: x >= y
-
-            Checks if a notification level is higher or equal to another.
-        .. describe:: x <= y
-
-            Checks if a notification level is lower or equal to another.
-    """
-
     all_messages = 0
-    """Members receive notifications for every message regardless of them being mentioned."""
     only_mentions = 1
-    """Members receive notifications for messages they are mentioned in."""
 
 
 class AuditLogActionCategory(int, Enum):
-    """Represents the category that the :class:`AuditLogAction` belongs to.
```

Comment:
restore

Reply:
Resolved in 606392a3f3350c8f1e22fa43c4b268855841d5cf.